pkg/system/init_windows.go

```diff
@@ -3,6 +3,7 @@ package system // import "github.com/docker/docker/pkg/system"
 import (
 	"os"
 
+	"github.com/Microsoft/hcsshim/osversion"
```

Comment:
oh! needs a re-run of `vndr` as they're not yet in `vendor`

```
16:44:59 INFO: Building daemon...
16:45:05 ..\..\pkg\system\init_windows.go:6:2: cannot find package "github.com/Microsoft/hcsshim/osversion" in any of:
16:45:05 	c:\go\src\github.com\docker\docker\vendor\github.com\Microsoft\hcsshim\osversion (vendor tree)
16:45:05 	C:\go\src\github.com\docker\docker\vendor\github.com\Microsoft\hcsshim\osversion
16:45:05 	c:\go\src\github.com\Microsoft\hcsshim\osversion (from $GOROOT)
16:45:05 	C:\go\src\github.com\Microsoft\hcsshim\osversion (from $GOPATH)
```

Reply:
Or a merge of my PR 😇

Reply:
Fixed